editor/modes/visual-editor/style.scss

```diff
@@ -77,24 +74,35 @@
 		background-color: rgba( $white, 0.6 );
 	}
 
-	&.is-hovered:before {
-		outline: 1px solid $light-gray-500;
-		transition: 0.2s outline;
+	// simpler style for a block that has cursor focus (but hasn't been selected)
+	&.is-selected .editor-block-mover:before,
```

Comment:
We may want to unify the className used for the `hover` and `selected`, maybe something like `is-ui-visible`